test/src/xquery/axes.xml

```diff
@@ -153,7 +153,7 @@
     <test output="text">
         <task>Following axis test: node()</task>
         <code>count(doc("/db/axes-test/axes1.xml")//line/word[. = 'working']/following::node())</code>
-        <expected>9</expected>
+        <expected>17</expected>
```

Comment:
so, default behavior to keep all WS and that's why 8 text nodes added here, right?